packages/openssl/windows.buildinfo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "sources": {
+    "openssl": {
+      "kind": "url",
+      "url": "https://slproweb.com/download/Win64OpenSSL-1_1_1e.exe",
+      "sha1": "b0b2fa45fd4e4ab6b502495bd2d706b570196911"
```

Comment:
Please use the `downloads.mesosphere` link from https://github.com/mesosphere/dcos-enterprise/blob/345157bc333e0ded07f8938f1d5c1cad83d99994/packages/mesos/windows-ee.buildinfo.json#L21-L25

Reply:
updated